courses/tasks.py

```diff
@@ -73,3 +78,17 @@ def exception_logging_generator(generator):
             log.exception("EdX API error for fetching user grades %s:", exc)
         except Exception as exp:  # pylint: disable=broad-except
             log.exception("Error fetching user grades from edX %s:", exp)
+
+
+@app.task
+def sync_courseruns_data():
```

Comment:
as it task, would you like to add some tests for that ?

Reply:
Done